node/bft/src/sync/mod.rs

```diff
@@ -471,10 +481,15 @@ impl<N: Network> Sync<N> {
             .filter_map(|k| latest_block_responses.get(&k).cloned())
             .collect();
 
-        // Check if the block response is ready to be added to the ledger.
-        // Ensure that the previous block's leader certificate meets the availability threshold
-        // based on the certificates in the current block.
-        // If the availability threshold is not met, process the next block and check if it is linked to the current block.
+        // Check if each block response, from the contiguous sequence just constructed,
+        // is ready to be added to the ledger.
+        // Ensure that the block's leader certificate meets the availability threshold
+        // based on the certificates in the DAG just after the block's round.
+        // If the availability threshold is not met,
+        // process the next block and check if it is linked to the current block,
+        // in the sense that there is a path in the DAG
```

Comment:
We currently set rustfmt to 120 character lines IIRC. So you can probably reduce the amount of line breaks you have in these comments, or is this how rustfmt formatted it?

Reply:
I normally put line breaks according to sentence "structure", since I find that easier to read and write, and it helps minimize diffs if there's any future change (e.g. removing a word and reformatting text makes it look like a much bigger change). But I can conform to a different convention, if there's agreement on one.